File: app/src/main/java/com/theappsolutions/boilerplate/data/model/api/ProjectsResponse.java
package com.theappsolutions.boilerplate.data.model.api;

import com.google.gson.annotations.SerializedName;

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
public class ProjectsResponse {

    @SerializedName("id")
    private long id;

    @SerializedName("name")
    private String name;

    @SerializedName("type")
    private String type;

    @SerializedName("logoSrc")
    private String logoSrc;

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public String getLogoSrc() {
        return logoSrc;
    }
}

File: app/src/main/java/com/theappsolutions/boilerplate/data/model/api/ProjectsResponse.kt
package com.theappsolutions.boilerplate.data.model.api

import com.google.gson.annotations.SerializedName

/**
 * @author Severyn Parkhomenko s.parkhomenko@theappsolutions.com
 * @copyright (c) 2018 TheAppSolutions. (https://theappsolutions.com)
 */
data class ProjectsResponse(
        @SerializedName("id") val id: Long,
        @SerializedName("name") val name: String,
        @SerializedName("type") val type: String,
        @SerializedName("logoSrc") val logoSrc: String
)